zeppelin-interpreter-integration/src/main/test/org/apache/zeppelin/integration/ZeppelinSparkClusterTest.java

```diff
@@ -225,7 +225,31 @@ public void sparkSQLTest() throws IOException {
     assertTrue(p.getReturn().message().get(0).getData().contains(
         "Array[org.apache.spark.sql.Row] = Array([hello,20])"));
 
-    // test display DataFrame
+    @Test
+    public void sparkReadCSVTest() throws IOException {
+      Note note = ZeppelinServer.notebook.createNote(anonymous);
+      Paragraph p = note.addNewParagraph(anonymous);
+      p.setText("%spark val csvStr = \"\"\"84896,54\"\"\"\n" +
+              "spark.read.csv(Seq(csvStr).toDS)");
+      note.run(p.getId(), true);
+      assertEquals(Status.FINISHED, p.getStatus());
+      assertTrue(p.getResult().message().get(0).getData().contains(
+              "org.apache.spark.sql.DataFrame = [_c0: string, _c1: string]\n"));
+    }
+
+    @Test
+    public void sparkSQLTest() throws IOException {
```

Comment:
Should be sparkReadJSONTest

Reply:
I forgot the JSON test and copied the wrong test. I will commit to this pull request. In fact, there should be 2 tests one for csv and one for json.